### PR TITLE
Add dark mode and minor adjustments to the color theme to match C4DT's logo

### PR DESCRIPTION
**Pull request by LucasTrg**
photo_2021-03-29_10-11-38

### CODE REVIEW

**Comment by LucasTrg** [comment]
In order to see the dark mode, you need to set your browser preferences to dark, I believe it's supported on most modern browsers (tested on IEE, Firefox and Chrome). We could perhaps add a radio button to toggle back and forth

**Comment by ineiti** [comment]
Great work. Can you please adjust the following:
- make the breadcrumb bar lighter, so it is more visible
- make the tags darker, so they stand less out

File: resources/styles.css [hunk 1]
```diff
@@ -71,80 +85,80 @@ table.dataTable tbody tr td.proj_name {
   font-weight: bold;
 }
 
-table.dataTable.display tbody tr.active.odd {
```
Comment (cgrigis) [inline resources/styles.css:74]: If I understand correctly, this removes the special `active` class.
Granted, it's not very clear :-) , but this class is actually [used](https://github.com/c4dt/showcase/blob/main/views/projects.tpl#L190) to mark the "active" projects in green (i.e. the ones whose last commit is within the last 9 months). On https://showcase.c4dt.org/, you can see that the projects towards the bottom should be mostly grey. That is unfortunately what makes the table CSS classes a bit cumbersome, as there are the alternate greys (even/odd) and the alternate greens (even/odd), in addition to the sorting versions...

**Review by cgrigis — CHANGES_REQUESTED** [review]

**Comment by ineiti** [comment]
I like the basic colors, however:
- the colors for the light mode should stay like in `main`
- when hovering over an active project, the color should stay something in the blue tone
- the tags background is still too strong - it should be darker, eventually even so dark that you need a white font
- @cgrigis changed a lot how things are done in the showcase in `main` - so you need to rebase it on that, sorry...

**Comment by LucasTrg** [comment]
Thanks to @tharvik, the branch is now rebased on main properly. I decided to discard the orange for the labels as I did not find any good way to make an orange rectangle on a dark background not eye-catching. Let me know what you think. Hopefully, this commit is the last one!

**Review by cgrigis — CHANGES_REQUESTED** [review]
I like the non-orange tags color in dark-mode!

Two remarks:
- The "active" projects in normal-mode are still not showing for me (tested on Firefox & Chromium)
- [Minor] The legend for the "active" projects is maybe a bit too close to the C4DT logo.

**Comment by LucasTrg** [comment]
Indeed, I may have messed up a commit while rebasing, I believe it's fixed now. I'm not 100% happy with how the legend looks in light mode, I might need to exaggerate the colors of the square because I can barely distinguish them (although that might be me)

**Comment by ineiti** [comment]
Very nice, thank you. As a last step, I would increase a little bit the intensity of the `hover` part, it's barely visible.

**Comment by LucasTrg** [comment]
Agreed ! The highlight is a bit bolder and more contrasting now

**Comment by ineiti** [comment]
For dark mode I think it makes more sense to lighten the hover: #177

**Comment by cgrigis** [comment]
> I believe it's fixed now.

Yes, thank you! :)

> I'm not 100% happy with how the legend looks in light mode, I might need to exaggerate the colors of the square because I can barely distinguish them (although that might be me)

Agreed. Perhaps a think black border around the colors?

**Review by ineiti — CHANGES_REQUESTED** [review]
need to solve conflict first

**Review by cgrigis — APPROVED** [review]

**Review by ineiti — APPROVED** [review]

**Pull request merged**